src/index.ts

```diff
@@ -67,6 +68,10 @@ async function renameBinary(
 
 async function getVersion(): Promise<string> {
   const version = core.getInput('version');
+  if (version === 'latest' || version === 'edge') {
+    return version;
+  }
```

Comment:
Reading the code again, the warning had misled me into believing it didn't work. So this function worked with "latest" before, and would work with "edge" now; adding the block here just gets rid of the warning. WDYT?